### PR TITLE
*: rm old region

### PR DESCRIPTION
**Pull request by disksing**
PTAL @ngaut @morefreeze

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: txn.go [hunk 1]
```diff
@@ -345,12 +337,12 @@ func (txn *tikvTxn) StartTS() int64 {
 
 type prewriteRegion struct {
 	*pb.CmdPrewriteRequest
-	c    *Client
-	raft *raftRegion
+	c      *Client
+	region *Region
 }
 
 type commitRegion struct {
```
Comment (morefreeze) [inline txn.go]: This can be removed.

**Comment by disksing** [comment]
PTAL @morefreeze

**Comment by morefreeze** [comment]
LGTM

**Pull request merged**